flavors/template-Exasol-all-python-3.10-conda/flavor_base/build_deps/packages/conda_packages

```diff
@@ -1,8 +1,6 @@
 protobuf|4.25.3
 pcre|8.45
-chrpath|0.16
```

Comment:
chrpath comes from us, I think

Reply:
However, build and tests passed without

Reply:
Added it back to conda and cuda-conda flavors